docs/guides/core-concepts/introduction-to-cypress.mdx

```diff
@@ -1097,17 +1079,17 @@ fail! Here's a few:
 
 <strong>Core Concept</strong>
 
-With Cypress, you don't have to assert to have a useful test. Even without
-assertions, a few lines of Cypress can ensure thousands of lines of code are
-working properly across the client and server!
+With Cypress, you don't have to write explicit assertions to have a useful test.
+Without a single `expect()` or `.should()`, a few lines of Cypress can ensure
+thousands of lines of code are working properly across the client and server.
 
 This is because many commands have a built in
-[Default Assertion](#Default-Assertions) which offer you a high level of
-guarantee.
+[Implicit Assertions](#Implicit-Assertions) which offer you a high level of
```

Comment:
feels like overkill to link to the next section.

Reply:
True. Removing link.